openslides/assignments/static/js/assignments/site.js

```diff
@@ -408,14 +427,23 @@ angular.module('OpenSlidesApp.assignments.site', ['OpenSlidesApp.assignments'])
     '$state',
     'Assignment',
     'AssignmentForm',
+    'Agenda',
     'assignment',
-    function($scope, $state, Assignment, AssignmentForm, assignment) {
+    function($scope, $state, Assignment, AssignmentForm, Agenda, assignment) {
+        Assignment.loadRelations(assignment, 'agenda_item');
         $scope.alert = {};
         // set initial values for form model by create deep copy of assignment object
         // so list/detail view is not updated while editing
         $scope.model = angular.copy(assignment);
         // get all form fields
         $scope.formFields = AssignmentForm.getFormFields();
+        for (var i = 0; i < $scope.formFields.length; i++) {
+            if ($scope.formFields[i].key == "showAsAgendaItem") {
+                // get state from agenda item (hidden/internal or agenda item)
+                // TODO: fix error after reload ("assignment.agenda_item is undefined")
```

Comment:
@ostcar How can I fix this error? I already add `Assignment.loadRelations(assignment, 'agenda_item');`... If I go to agenda and then back to AssignmentForm it works.